### PR TITLE
refresh wallet account details on display currency change

### PR DESCRIPTION
**Pull request by zanderz**

### CODE REVIEW

**Comment by zanderz** [comment]
not sure what that might mean except to refresh instead of selecting?

**Comment by zanderz** [comment]
ah ok I think I see

File: shared/actions/wallets.js [hunk 1]
```diff
@@ -778,6 +784,7 @@ function* walletsSaga(): Saga.SagaGenerator<any, any> {
   yield Saga.actionToPromise(WalletsGen.loadDisplayCurrencies, loadDisplayCurrencies)
   yield Saga.actionToPromise(WalletsGen.loadSendAssetChoices, loadSendAssetChoices)
   yield Saga.actionToPromise(WalletsGen.loadDisplayCurrency, loadDisplayCurrency)
+  yield Saga.actionToAction(WalletsGen.loadDisplayCurrency, refreshAssets)
```
Comment (chrisnojima) [inline shared/actions/wallets.js]: we want 
DisplayCurrenciesReceived and not loadDisplayCurrency

Comment (zanderz) [reply shared/actions/wallets.js]: ok thanks

**Pull request merged**